cratedb_django/fields/__init__.py

```diff
@@ -53,8 +56,19 @@ class FloatField(CrateDBBaseField, fields.FloatField):
     pass
 
 
-class GeneratedField(CrateDBBaseField, fields.FloatField):
-    pass
+class GeneratedField(CrateDBBaseField, GeneratedField):
+    def __init__(self, *args, **kwargs):
+        virtual = kwargs.get("db_persist")
+        if virtual:
+            logging.warning(
+                f"{self} has virtual=True, but CrateDB does not support virtual generated columns."
```

Comment:
I am not sure about this. While I see you are storing the outcome into a variable `virtual` (isn't that the semantic opposite to "persist"?), I can't spot the notion of `virtual=True` in any user space or testing code, so there is a chance this is just a typo?
```suggestion
                f"{self} has db_persist=True, but CrateDB does not support virtual generated columns."
```